app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -8,13 +8,15 @@ import org.wikipedia.dataclient.RestService
 import org.wikipedia.page.Namespace
 import org.wikipedia.page.PageViewModel
 import org.wikipedia.settings.Prefs
+import org.wikipedia.util.DimenUtil.getDensityScalar
+import org.wikipedia.util.DimenUtil.leadImageHeightForDevice
 import org.wikipedia.util.L10nUtil
 import org.wikipedia.util.L10nUtil.formatDateRelative
 
 object JavaScriptActionHandler {
     @JvmStatic
     fun setHandler(): String {
-        return "pcs.c1.InteractionHandling.setInteractionHandler((interaction) => { marshaller.onReceiveMessage(JSON.stringify(interaction))})"
+        return "pcs.c1.InteractionHandling.setInteractionHandler((interaction) => { pcsClient.onReceiveMessage(JSON.stringify(interaction))})"
```

Comment:
@sharvaniharan it should be possible to remove `setHandler()` - PCS should call `onRecieveMessage` on `pcsClient` directly without having to call `setInteractionHandler`

Reply:
Done. Thanks @joewalsh